Sources/CloudFoundryEnv/AppEnv.swift

```diff
@@ -177,7 +177,7 @@ extension ConfigurationManager {
     }
 
     do {
-      #if os(Linux)
+      #if os(Linux) && !swift(>=3.1)
```

Comment:
@youming-lin:  After discussing this issue in more depth, we have decided reject this change.  While version-1 backwards compatibility will be very important moving forward, we are reluctant to set this precedent until the TSC issues an edict for how all teams should address this.